server/handlers/isomorphic-handler.js

```diff
@@ -412,12 +392,7 @@ exports.handleIsomorphicRoute = function handleIsomorphicRoute(
     const seoInstance = getSeoInstance(seo, config, result.pageType);
     const seoTags =
       seoInstance &&
-      seoInstance.getMetaTags(
-        config,
-        result.pageType || match.pageType,
-        result,
```

Comment:
`|| match.pageType` removed as `match` was not defined on this page. Was this an intentional global?

Reply:
or should I move `    const match = matchBestRoute(url.pathname, staticRoutes);` here?